src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -441,6 +441,9 @@ public void testVersion() throws Exception {
     assertThat(version.kernelVersion(), not(isEmptyOrNullString()));
     assertThat(version.os(), not(isEmptyOrNullString()));
     assertThat(version.version(), not(isEmptyOrNullString()));
+    if (dockerApiVersionAtLeast("1.22")) {
+      assertThat(version.buildTime(), not(isEmptyOrNullString()));
```

Comment:
this needs to be updated (it fails to compile) since `version.buildTime()` is a Date not a String